### DISCUSSION TITLE
Are there circumstances in which an overlay tool could be seen as providing a conforming alternate version for a non-conforming website?

### DISCUSSION THREAD

**Discussion by detlevhfischer** [w3c/wcag#2749] [Q&A]
Public bodies increasingly fall for promises of accessibility overlay providers that putting an overlay on their sites will make them WCAG 2.1 conformant in a matter of hours. In all cases we know of this is not true, and the promise grossly misleading. Moreover, most overlay tools we have investigated have themselves some or many accessibility barriers in the settings dialog offered to users for customisation of the site. But this is not the topic I would like the WG to clarify here.

This question is this: 
**In the (unlikely) case that an overlay tool is actually able to successfully remedy all WCAG fails of a site via a specific set of user settings, can this state be considered a conforming alternate version (CAV)?**

Looking at the Conformance requirements and the definition of CAV, the situation seems a bit fuzzy.

Section [5.2. Conformance Requirements](https://www.w3.org/TR/WCAG21/#conformance-reqs) talks about one specific version ("…or a conforming alternate version is provided")  and [the definition of conforming alternate version](https://www.w3.org/TR/WCAG21/#dfn-conforming-alternate-version) also talks about "the conformaing alternative version" (singular) . One note explicitly qualifiies "One version would need to be fully conformant in order to meet conformance requirement 1."

Where it gets fuzzy is with [the last note of the CAV definition](https://www.w3.org/TR/WCAG21/#h-note-72), which says:
> "Setting user preferences within the content to produce a conforming version is an acceptable mechanism for reaching another version as long as the method used to set the preferences is accessibility supported."

There is no stipulation that just **one** switch (say, a style switcher typically used to load a more contrasty version of a page) or **one** link to a CAV is mandated. However, I would argue that without an explicit label or an explicit description in the overlay interface what settings users would need to change to produce the CAV, the mechanism cannot be considered accessibility supported (arguably failing [2.4.6 Headings and Labels](https://www.w3.org/TR/WCAG21/#headings-and-labels)).

If this argument is the position the Working Group would adopt, it would follow that **a generic overlay without some explicit signposting how to reach the CAV can never be considered a CAV** even in the (very unlikely) case that the overlay would be able to remedy **all** WCAG failures of a site with some combination of settings.

**Comment by patrickhlauke** [comment]
Moved this to a discussion, as it seems less about WCAG/updating something, and more an initial general question

**Reply by detlevhfischer** [reply]
@patrickhlauke Consensus could lead to a normative update of Section 5.2 Conformance. This is far less visible as a discussion. I think it would be important to reach working group consensus on this - I think a Working Group reply should go into a survey to establish a position on this matter (or else show that there is no consensus). So I think it should remain an issue to be solved, not a discussion.

**Comment by patrickhlauke** [comment]
ok, the group probably needs to decide how/if/when they even want to use discussion features. at the moment, it seems a bit ad-hoc either way (though looking at the fact there's 585 open issues - and 191 open pull requests - on WCAG at the moment...don't think the group will ever manage to get a handle on the situation)

**Reply by detlevhfischer** [reply]
@patrickhlauke you say ok - so can you revert to issue? I see your point about the workload, but this issue seems pressing (to be able to point public bodies enquiring about this to a WG position) and could potentially be a quick and near unanimous decision.

**Reply by patrickhlauke** [reply]
> you say ok - so can you revert to issue?

I believe you can do it as well, but sure

**Reply by patrickhlauke** [reply]
https://github.com/w3c/wcag/issues/2750

**Reply by detlevhfischer** [reply]
I had seen that option but wasn't sure whether you woud then revert it back to discussion... Thanks anyway.

**Comment by yatil** [comment]
[I commented on the issue](https://github.com/w3c/wcag/issues/2750#issuecomment-1291614960). Maybe close this discussion with a big link for redirecting people.
